### PR TITLE
There were multiple variants of this in Cromwell and Lenthall, reduce…

### PR DESCRIPTION
**Pull request by geoffjentry**
… to a single one

### CODE REVIEW

**Comment by geoffjentry** [comment]
@mcovarr and @Horneth - the wheel is awfully thin today

**Comment by Horneth** [comment]
:+1:

File: src/main/scala/lenthall/config/ScalaConfig.scala [hunk 1]
```diff
@@ -1,52 +1,30 @@
...
+    def getBooleanOr(key: String, default: => Boolean = false): Boolean = getBooleanOption(key) getOrElse default
+    def getIntOr(key: String, default: => Int = 0): Int = getIntOption(key) getOrElse default
+    def getLongOr(key: String, default: => Long = 0L): Long = getLongOption(key) getOrElse default
+    def getDoubleOr(key: String, default: => Double = 0.0): Double = getDoubleOption(key) getOrElse default
+
+    private def getOption[T](key: String, f: String => T): Option[T] = {
+      Try(f(key)) match {
+        case scala.util.Success(value) => Option(value)
+        case scala.util.Failure(e: ConfigException.Missing) => None
+        case scala.util.Failure(e) => throw e
```
Comment (mcovarr) [inline src/main/scala/lenthall/config/ScalaConfig.scala]: why the FQNs?

**Comment by mcovarr** [comment]
:+1:

**Pull request merged**